starter-core/src/main/java/io/micronaut/starter/feature/build/MicronautBuildPlugin.java

```diff
@@ -50,7 +55,14 @@ public String getName() {
     @Override
     public void apply(GeneratorContext generatorContext) {
         if (generatorContext.getBuildTool().isGradle()) {
-            generatorContext.addBuildPlugin(shouldApplyMicronautApplicationGradlePlugin(generatorContext) ?
+
+            boolean shouldApplyApplicationPlugin = shouldApplyMicronautApplicationGradlePlugin(generatorContext);
+
+            generatorContext.addHelpLink("Micronaut Gradle Plugin documentation", MICRONAUT_GRADLE_DOCS_URL);
+            if (shouldApplyApplicationPlugin && (generatorContext.getLanguage() == JAVA || generatorContext.getLanguage() == KOTLIN)) {
```

Comment:
I think the library plugin applies also the graalVM gradle plugin. Thus I don't think we need to check for `shouldApplyApplicationPlugin` to add the docs